js/renovation/ui/scroll_view/__tests__/scrollable.test.tsx

```diff
@@ -42,17 +42,36 @@ jest.mock('../../../../core/utils/scroll_rtl_behavior', () => () => testBehavior
   describe('Scrollable', () => {
     describe('Render', () => {
       it('should render scrollable content', () => {
-        const scrollView = shallow(viewFunction({ props: { } } as any) as JSX.Element);
-        const scrollViewContent = scrollView.find('.dx-scrollable-wrapper .dx-scrollable-container .dx-scrollable-content');
-        expect(scrollViewContent.exists()).toBe(true);
+        const scrollable = shallow(viewFunction({ props: { } } as any) as JSX.Element);
+        const scrollableContent = scrollable.find('.dx-scrollable-wrapper .dx-scrollable-container .dx-scrollable-content');
+        expect(scrollableContent.exists()).toBe(true);
+      });
+
+      it('should not render top & bottom pockets', () => {
+        const scrollable = shallow(viewFunction({ props: { } } as any) as JSX.Element);
+        const scrollableTop = scrollable.find('.dx-scrollable-wrapper .dx-scrollable-container .dx-scrollable-content > .dx-scrollview-top-pocket');
+        expect(scrollableTop.exists()).toBe(false);
+        const scrollableBottom = scrollable.find('.dx-scrollable-wrapper .dx-scrollable-container .dx-scrollable-content > .dx-scrollview-bottom-pocket');
+        expect(scrollableBottom.exists()).toBe(false);
+      });
+
+      it('should render top & bottom pockets', () => {
+        const scrollable = shallow(viewFunction({
+          props:
+          { forceGeneratePockets: true },
+        } as any) as JSX.Element);
+        const scrollableTop = scrollable.find('.dx-scrollable-wrapper .dx-scrollable-container .dx-scrollable-content > .dx-scrollview-top-pocket');
```

Comment:
Again, what about exact tree?

Reply:
Change to direct child?
const scrollableTop = scrollable.find('.dx-scrollable-wrapper .dx-scrollable-container .dx-scrollable-content > .dx-scrollview-top-pocket');

Reply:
Fixed in https://github.com/DevExpress/DevExtreme/pull/15548/commits/81c0c59a38bb62e1573599a33b088386786b0eaf